.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -2,18 +2,23 @@
   <profile version="1.0">
     <option name="myName" value="Project Default" />
     <inspection_tool class="PreviewAnnotationInFunctionWithParameters" enabled="true" level="ERROR" enabled_by_default="true">
+      <option name="composableFile" value="true" />
```

Comment:
seems like this file should be getting ignored 🤔 https://github.com/appcues/appcues-android-sdk/blob/main/.gitignore#L22

Reply:
weird looking into it.

Reply:
it looks like gitIgnore ignores newly added files, but it keep tracks of existing ones, I will try to push a deletion of this one so it doesnt bother us again.